guides/source/layouts_and_rendering.md

```diff
@@ -1266,7 +1266,7 @@ You can also pass in arbitrary local variables to any partial you are rendering
 
 In this case, the partial will have access to a local variable `title` with the value "Products Page".
 
-TIP: Rails also makes a counter variable available within a partial called by the collection, named after the member of the collection followed by `_counter`. For example, if you're rendering `@products`, within the partial you can refer to `product_counter` to tell you how many times the partial has been rendered. This does not work in conjunction with the `as: :value` option.
+TIP: Rails also makes a counter variable available within a partial called by the collection, named after the title of the partial followed by `_counter`. For example, if you're rendering `@products` with a partial `_product.html.erb', within the partial you can refer to `product_counter` to tell you how many times the partial has been rendered. This does not work in conjunction with the `as: :value` option.
```

Comment:
Thanks for the PR! I'm finding this part confusing and long winded

>  if you're rendering `@products` with a partial `_product.html.erb', within the partial you can refer to `product_counter` to tell you how many times the partial has been rendered

What do you think about this instead?

> if you're rendering `@products` with the `_product.html.erb' partial you can refer to `product_counter` to tell you how many times the partial has been rendered within the partial.

Reply:
Hi Eileen,

I guess what I was trying to be clear about is that the variable `product_counter` would be available within the `_product.html.erb` partial, and not the enclosing view (which may or may not be another partial, right?). The way I read your version, by moving the phrase 'within the partial' from where I put it to the end of the sentence you've changed the meaning slightly so that the second 'the partial' now means means the enclosing view.

I guess this might be clear from the context, but I was trying to avoid confusion with ruby constructs that make an index available such as  
```ruby 
array.each_with_index | value, index|
#do stuff that knows about array[ index] = value
end
```
where you might imagine that `product_counter` was made available in the calling scope.

Of course, you wouldn't be the first person to accuse me of being long-winded ...
How about this? 
> For example, when rendering a collection `@products`  the partial `_product.html.erb` can access the variable `product_counter` that indexes the number of times it has been rendered within the enclosing view.

Reply:
Hey @robwold yea I think that's better. Can you make that change and squash your commits into one?

Reply:
Done. Let me know if there's any further issues.